doc/source/ray-air/benchmarks.rst

```diff
@@ -143,6 +143,10 @@ We test out the performance across different cluster sizes and data sizes.
 
 - `GPU image training script`_
 
+.. note::
+
+    For multi-host distributed training, on AWS we need to ensure ec2 instances are in the same VPC, and all ports are open in the secure group.
```

Comment:
also they should also be in the same placement group?

Reply:
yes that's what i had in my workspaces too. Do you know any train's existing tests that had the right YAML setup for this ?

Reply:
Yes this tests is run with multi-host GPU training: https://github.com/ray-project/ray/blob/master/release/release_tests.yaml#L925-L947

Reply:
hmm that seems to point to a CPU compute template. Let me create a public PlacementGroup and secure group on CI

Reply:
This is the compute template that it's using https://github.com/ray-project/ray/blob/master/release/ml_user_tests/train/compute_tpl.yaml

Uses 3 g3.8xlarge isntances

Reply:
right, i didn't find any config regarding their secure group or placement group tho, am i missing something here or it's not needed for our previous multi-host gpu tests

Reply:
Ah that's right- what I'm saying is I don't think it's needed if we set the runtime env

Reply:
i see what you mean now, makes sense and let me update the PR :)